docs/apidiffs/current_vs_latest/opentelemetry-extension-annotations.txt

```diff
@@ -1,2 +1,10 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
++++  NEW ANNOTATION: PUBLIC(+) ABSTRACT(+) io.opentelemetry.extension.annotations.SpanAttribute  (not serializable)
```

Comment:
What's with the changes in these files? I'm getting the same diff locally if I do a totally unrelated change (presumably also if I just rebuild).

Probably this needs to go into a separate PR and was missing from some other PR.

Reply:
Yeah I've been noticing this. I think the two reasonable options are

1) Fail PR build if there are API diffs
2) Move diff computation and creation of a PR to a nightly job and remove it from normal `check`.

I like 2) better, but it requires a PAT which still can't be scoped 😿

I don't think requiring contributors to split their PRs or deal with merge conflicts because the tooling isn't ideal yet is a good idea though so wouldn't worry too much about it until doing one of those.